kornia/augmentation/base.py

```diff
@@ -141,15 +154,15 @@ def identity_matrix(self, input: torch.Tensor) -> torch.Tensor:
     def compute_transformation(self, input: torch.Tensor, params: Dict[str, torch.Tensor]) -> torch.Tensor:
         raise NotImplementedError
 
-    def __infer_input__(  # type: ignore
+    def __unpack_input__(  # type: ignore
```

Comment:
why type ignore here ?

Reply:
Our very base class implemented this for tensor only input, while the inherited base (this one) would accept Union type.